README.md

```diff
@@ -1,75 +1,72 @@
-# QHub
+# `qhub` - automated data science environments on cloud environments
 
-Is a tool for initializing and maintaining the state of QHub
-deployments on Digital Ocean, Amazon Web Services, and Google Cloud
-Platform.
+:badges:
 
-## Installation:
+[`qhub`][qhub] is an opinionated deployment of open source data science infrastrcuture (eg. [jupyterhub], [dask], and [kubernetes]) on different cloud service providers without vendor lock-in. [`qhub`][qhub] is easy to install and maintain.
 
-```bash
-pip install qhub
-```
+## Usage
 
-## Configuration
+`qhub` is install as a command line application in python. it requires you to choose your the cloud provider you desire.
 
-QHub is entirely controlled from a configuration file. 
+        qhub init do
+        qhub init aws
+        qhub init gcp
 
-To generate a configuration file follow these instructions.
+The `qhub init` command will generate configuration files for that service. The configuartion files can be tailored to the needs of your organization. Each file specifies general project information, security, infrastracture settings, computational resource profiles, and data science environments. See documentation on modifying your configuration file for all of the cloud providers: [Configuration File](https://github.com/Quansight/qhub/blob/master/docs/docs/aws/configuration.md) 
```

Comment:
The configuartion -> The The configuration
infrastracture -> infrastructure
profiles, and -> profiles and